airbyte-metrics/reporter/src/main/java/io/airbyte/metrics/reporter/MetricRepository.java

```diff
@@ -36,7 +41,13 @@ Map<String, Integer> numberOfPendingJobsByGeography() {
         .on(CONNECTION.ID.cast(VARCHAR(255)).eq(JOBS.SCOPE))
         .where(JOBS.STATUS.eq(JobStatus.pending))
         .groupBy(CONNECTION.GEOGRAPHY);
-    return (Map<String, Integer>) result.fetchMap(0, 1);
+    var queriedMap = (Map<String, Integer>) result.fetchMap(0, 1);
```

Comment:
nit: instead of casting can we leverage the `fetchMap` function to explicitly type transform for us?

Reply:
Done!